deepchecks/tabular/checks/performance/confusion_matrix_report.py

```diff
@@ -11,17 +11,30 @@
 """The confusion_matrix_report check module."""
 import numpy as np
 import pandas as pd
-import plotly.express as px
-import sklearn
+from sklearn import metrics
 
 from deepchecks.core import CheckResult
 from deepchecks.tabular import Context, SingleDatasetCheck
+from deepchecks.utils.plot import create_confusion_matrix_figure
+
 
 __all__ = ['ConfusionMatrixReport']
 
 
 class ConfusionMatrixReport(SingleDatasetCheck):
-    """Calculate the confusion matrix of the model on the given dataset."""
+    """Calculate the confusion matrix of the model on the given dataset.
+
+    Parameters
+    ----------
+    is_normalize (bool, default True):
+        If to color the matrix by normalizing by true values.
```

Comment:
this sentence is not clear

Reply:
agree

Reply:
can you give me a better option?

Reply:
"boolean that determines whether to normalize the true values of the matrix"?
Also, maybe "should_normalize" or just "normalize"? I dislike "is_..." format.